algebird-test/src/test/scala/com/twitter/algebird/macros/CuberMacroTest.scala

```diff
@@ -0,0 +1,67 @@
+package com.twitter.algebird.macros
+
+import com.twitter.algebird._
+import com.twitter.algebird.macros._
+import com.twitter.algebird.macros.ArbitraryCaseClassMacro.arbitrary
+
+import org.scalacheck.{ Properties, Arbitrary }
+import org.scalacheck.Prop.forAll
+
+object CuberRollerProperties extends Properties("Cuber and roller macro") {
```

Comment:
can we use scalatest here instead of scalacheck so it runs with the other tests? It will get counted towards the metrics then of how many tests we've ran overall